data/src/main/java/org/apache/iceberg/data/GenericAppenderFactory.java

```diff
@@ -47,27 +47,31 @@
  */
 public class GenericAppenderFactory implements FileAppenderFactory<Record> {
```

Comment:
We have just recently introduced `WriterFactory`. We will be moving to that API instead of appenders.
I think it will be sufficient to just cover the new API. Plus, it should be super simple as I thought about this change while implementing `BaseWriterFactory`. It already has a reference to `Table`. This will help us to avoid changing so many places around `AppenderFactory` that we will remove in the nearest future.

Reply:
Done, removed changes to unnecessary files